pointcloud/basic_octree_utils_test.go

```diff
@@ -220,7 +220,7 @@ func validateBasicOctree(t *testing.T, bOct *BasicOctree, center r3.Vector, side
 		}
 		test.That(t, size, test.ShouldEqual, bOct.size)
 		test.That(t, bOct.node.maxVal, test.ShouldEqual, maxVal)
-		test.That(t, bOct.node.maxVal, test.ShouldEqual, bOct.MaxVal())
+		test.That(t, bOct.node.maxVal, test.ShouldEqual, bOct.MaxVal()) // TODO throwing error
```

Comment:
Whats this doing?

Reply:
oh my bad that's from my debugging notes